rules/autoconsent/fedex.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "fedex",
+    "vendorUrl": "https://www.fedex.com/en-us/home.html",
+    "cosmetic": false,
+    "prehideSelectors": ["fedex-gdpr"],
+    "detectCmp": [
+        {
+            "exists": "fedex-gdpr"
+        }
+    ],
+    "detectPopup": [
+        {
+            "visible": "fedex-gdpr div"
+        }
+    ],
+    "optIn": [
+        {
+            "waitForThenClick": "fedex-gdpr .fxg-gdpr__accept-all-btn"
+        }
+    ],
+    "optOut": [
+        {
+            "waitForThenClick": "fedex-gdpr .fxg-gdpr__reject-all-btn"
+        }
+    ],
+    "test": [
+        {
+            "cookieContains": "_svs="
+        }
```

Comment:
It looks like this check doesn't work from a US IP, but the popup is managed.

Reply:
the self-test worked for me, but I think there's a race condition between handling `optOutResult` and `autoconsentDone` messages in the test extension background page